spec/javascript/spec_helper.js

```diff
@@ -40,6 +40,8 @@ global.window.Response = Response;
 useCleanDOM(dom);
 useConsoleLogSpy();
 
+global.navigator.sendBeacon = () => true;
```

Comment:
The fake browser environment we're using for tests didn't have `navigator.sendBeacon` available, and Sinon wouldn't let me stub it unless it was there.

Reply:
Makes sense to me 👍 I may suggest moving up a few lines to keep together with all the other stubbing we have on the globals, e.g. after line 38.